crates/router/src/core/payments.rs

```diff
@@ -3614,17 +3711,31 @@ where
     if let Some(ref routing_algorithm) = routing_data.routing_info.algorithm {
         let (mut connectors, check_eligibility) = routing::perform_straight_through_routing(
             routing_algorithm,
-            payment_data.creds_identifier.clone(),
+            payment_data.get_creds_identifier(),
         )
         .change_context(errors::ApiErrorResponse::InternalServerError)
         .attach_printable("Failed execution of straight through routing")?;
 
         if check_eligibility {
+            // let setup_mandate = payment_data.get_setup_mandate();
```

Comment:
Same here

Reply:
Sure will remove these, the changes are addressed in this https://github.com/juspay/hyperswitch/pull/5851